project-api/project-api-test/src/test/resources/myproject_with_lang_import/modules/services/svc.bal

```diff
@@ -0,0 +1,6 @@
+import sameera/myproject.storage;
+
+
```

Comment:
```suggestion
```

Reply:
fixed in 274babad95104c166f871220329483954e88cf7b commit

Reply:
Let's resolve the comments for small fixes and keep only the ones we need further discussions